pegasus/sites.v3/code.org/views/minecraft_2022_promo.haml

```diff
@@ -0,0 +1,27 @@
+%link{href: "/css/tutorial-promo.css", rel: "stylesheet"}
+
+:ruby
+  mee_image = "/images/mc/fit-1940x620/mc_mee_wide_large_2022.png"
+  mee_title = I18n.t(:hoc2022_mc_mee_title)
+  mee_description = I18n.t(:hoc2022_mc_mee_description)
+  mee_link = "https://aka.ms/HourofCode2022"
```

Comment:
This link doesn't appear to work –– will it be updated?

Reply:
That's the link the Minecraft team gave us, and they'll flip it on when it's ready on their end. I'll hold off merging this PR until we know when that will be (I'll find this out).

Reply:
Got it, sounds good!